controllers/imagecollector/imagecollector_controller.go

```diff
@@ -233,75 +242,74 @@ func (r *Reconciler) createImageJob(ctx context.Context, req ctrl.Request, argsC
 	scanDisabled := *scannerImage == ""
 	startTime = time.Now()
 
-	job := &eraserv1alpha1.ImageJob{
-		ObjectMeta: metav1.ObjectMeta{
-			GenerateName: "imagejob-",
-			Labels: map[string]string{
-				util.ImageJobOwnerLabelKey: ownerLabelValue,
+	jobTemplate := corev1.PodTemplateSpec{
+		Spec: corev1.PodSpec{
+			Volumes: []corev1.Volume{
+				{
+					// EmptyDir default
+					Name: "shared-data",
+				},
 			},
-		},
-		Spec: eraserv1alpha1.ImageJobSpec{
-			JobTemplate: corev1.PodTemplateSpec{
-				Spec: corev1.PodSpec{
-					Volumes: []corev1.Volume{
-						{
-							// EmptyDir default
-							Name: "shared-data",
+			RestartPolicy: corev1.RestartPolicyNever,
+			Containers: []corev1.Container{
+				{
+					Name:            "collector",
+					Image:           *collectorImage,
+					ImagePullPolicy: corev1.PullIfNotPresent,
+					Args:            append(collectorArgs, "--scan-disabled="+strconv.FormatBool(scanDisabled)),
+					VolumeMounts: []corev1.VolumeMount{
+						{MountPath: "/run/eraser.sh/shared-data", Name: "shared-data"},
+					},
+					Resources: corev1.ResourceRequirements{
+						Requests: corev1.ResourceList{
+							"cpu":    resource.MustParse("7m"),
```

Comment:
are we hardcoding these?

Reply:
These have been hardcoded for some time, the code just moved

Reply:
is this part of the config we talked about before?

Reply:
It should be. I'll leave a link to this comment in the configmap issue.